src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -167,14 +167,15 @@ private void initUI() {
 
     private ScrollPane createAllMilestonesScrollPane() {
         VBox milestonesBox = new VBox();
-        milestonesBox.setStyle("-fx-border-radius: 3;-fx-background-color: white;-fx-border-color: black;");
+        milestonesBox.setPrefWidth(398);
```

Comment:
Does this currently affect the visibility of the milestones? If not, I think width changes should be handled in #1525 instead.